golos.publication/golos.publication.cpp

```diff
@@ -69,11 +70,15 @@ void publication::create_post(account_name account, std::string permlink,
         item.account = account;
         item.permlink = permlink_hash;
         item.parentacc = parentacc;
-        item.parentprmlnk = parentprmlnk;
+        item.parentprmlnk = get_checksum256(parentprmlnk);
         item.curatorprcnt = curatorprcnt;
         item.payouttype = payouttype;
         item.beneficiaries = beneficiaries;
         item.paytype = paytype;
+        if (parentacc) {
+            set_child_count(true, parentacc, parentprmlnk);
+        }
+        item.isclosed = POST_OPEN;
```

Comment:
`isclosed` is not good name for state, it's good for bool flag.
* rename it to something like `state`/`status`

or
* use `bool` type and remove `POST_OPEN`/`_CLOSE` defines

btw, there are no limitations on field names, you can safely use underscore (like `is_closed`)